packages/ag-charts-types/src/chart/axisOptions.ts

```diff
@@ -41,8 +41,6 @@ export interface AgAxisCaptionOptions {
 export interface AgBaseAxisOptions<LabelType = any> {
     /** Axis type identifier. */
     type: string;
-    /** An array of keys determining which series are charted on this axis. */
-    keys?: string[];
```

Comment:
@iMoses please flag this as a fixing breaking change to @AG-DavidG

Reply:
Is it a breaking change though?
It used to be on the base and then omitted from polar axes, now it moved to the cartesian axis base.
From a user/docs perspective nothing has changed.